keen/api.py

```diff
@@ -190,6 +211,45 @@ def delete_events(self, event_collection, params):
         self.error_handling(response)
         return True
 
+    def get_collection(self, collection_name):
+        """
+        Extracts info about a collection using the Keen IO API. A master key must be set first.
+
+        :param event_name: the name of the collection to retrieve info for
+        """
+        if not self.master_key:
+            raise exceptions.InvalidEnvironmentError(
+                "The Keen IO API requires a master key to get events. "
+                "Please set a 'master_key' when initializing the "
+                "KeenApi object."
+            )
+        url = "{0}/{1}/projects/{2}/events/{3}".format(self.base_url, self.api_version,
+                                                       self.project_id, collection_name)
```

Comment:
`collection_name` (also on line 214) should be `event_collection` for the sake of consistency.